zh/get-started.md

```diff
@@ -239,7 +239,7 @@ TiDB Operator 包含许多实现 TiDB 集群不同组件的自定义资源类型
 {{< copyable "shell-regular" >}}
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.7/manifests/crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.8/manifests/crd.yaml
```

Comment:
Please also update the version in `en/get-started.md`.

Reply:
fixed